gems/aws-sdk-dynamodbstreams/lib/aws-sdk-dynamodbstreams/customizations.rb

```diff
@@ -1,3 +1,4 @@
+# frozen_string_literal: true
```

Comment:
This file is code generated. Did you add it manually or update any code generation?

Reply:
Manually, which suggest I might have missed a template somewhere?

Reply:
Checking out the build logic - it looks like there is no view for customizations (https://github.com/aws/aws-sdk-ruby/blob/master/build_tools/aws-sdk-code-generator/lib/aws-sdk-code-generator/code_builder.rb#L61)

I think that this generated code warning has been committed but then later removed for customizations files. I erased the contents of the file and ran `rake build:aws-sdk-dynamodbstreams` and it remained as a blank file. I think you can erase the contents here.

Reply:
Done 🔥